app/assets/stylesheets/local/custom.scss

```diff
@@ -49,8 +49,17 @@
   }
 }
 
+$app-blue-button-colour: govuk-colour("blue");
+.app-govuk-button--blue {
```

Comment:
I think there's already a class for this ```app-button--inverted```

Reply:
I'm afraid it is not the same 😞 
The inverted will set the background colour to white. We need it blue.
It works well on the blue containers, but outside a blue container is very different:

<img width="556" alt="Screenshot 2022-10-31 at 15 07 18" src="https://user-images.githubusercontent.com/687910/199041534-51b91391-6ee8-423a-93c9-d903fbfb9949.png">

<img width="705" alt="Screenshot 2022-10-31 at 15 07 54" src="https://user-images.githubusercontent.com/687910/199041559-bf676054-03e7-4c55-a716-208c1fd17403.png">

However to follow the same naming pattern I think I will rename `app-govuk-button--blue` to `app-button--blue` 👍